src/extension.ts

```diff
@@ -114,7 +116,7 @@ export function activate(context: ExtensionContext) {
     }));
 
     context.subscriptions.push(commands.registerCommand(GaugeVSCodeCommands.ExecuteAllSpecExplorer, () => {
-        return runSpecification(specExplorerActiveFolder);
+        return runSpecification(treeDataProvider.workspaceRoot);
```

Comment:
the `workspaceRoot` is set by [`extention.ts`](https://github.com/getgauge/gauge-vscode/pull/202/files#diff-45327f86d4438556066de133327f4ca2R170) and again accessed here by the same `extension`. This has also caused `workspaceRoot` to be public in `SpecNodeProvider`. 

we can change this.

And perhaps we can use `specNodeProvider` instead of `treeDataProvider` as the name of the variable here.